src/Components/Box/box.js

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from 'react';
+import { UserContext } from '../../Context/GameContext.js';
+import './box.css';
+
+export default function Box({ space, content }) {
+  const { setSpace, turnHandler, checkSpace, checkGame, endGame } = useContext(UserContext);
+  const className = checkSpace(content);
+  const gameEnder = endGame(content);
+  return (
+    <div
+      className={`${className} ${gameEnder} box`}
+      onClick={() => {
+        setSpace(space);
+        turnHandler();
+        checkSpace(content);
+        checkGame();
```

Comment:
often you'll see all the click events abstracted out into a separate function called something like 'clickHandler' so you'll just see ```onClick={clickHandler}```